gwcli/ceph.py

```diff
@@ -256,10 +301,6 @@ def populate(self):
             name = pool['pool_name']
             pools[name] = pool
 
-        # # Get the pools defined
```

Comment:
While you are changing this -- perhaps hide cache tiers?

Reply:
I can add this to my to-do sure - but for this PR I just want to get the health issues fixed